Detectors/TPC/qc/include/TPCQC/Tracks.h

```diff
@@ -68,6 +68,18 @@ class Tracks
   void dumpToFile(std::string_view filename);
 
   // To set the elementary track cuts
+  void setTrackCuts(float AbsEta = 1.,
+                    int nClusterCut = 60, float dEdxTot = 20, float cutDCAr = 3, float cutPtForDCAr = 1.5, float samplingFractionDCAr = 0.1)
+  {
+    mCutAbsEta = AbsEta;
+    mCutMinnCls = nClusterCut;
+    mCutMindEdxTot = dEdxTot;
+    mCutMinDCAr = cutDCAr;
+    mCutMaxPtDCAr = cutPtForDCAr;
```

Comment:
Shouldn't this be a min pt cut? Should be clarified.
```suggestion
    mCutMinPtDCAr = cutPtForDCAr;
```